servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/H2SchemeTest.java

```diff
@@ -158,7 +157,7 @@ private SchemeValidatorHandler(final HttpScheme expectedScheme) {
         public void channelRead(final ChannelHandlerContext ctx, final Object msg) {
             if (msg instanceof Http2HeadersFrame) {
                 Http2Headers headers = ((Http2HeadersFrame) msg).headers();
-                if (!expectedScheme.name().equals(headers.scheme())) {
+                if (!expectedScheme.name().contentEquals(headers.scheme())) {
```

Comment:
To prevent a possible backward compatibility breaking, it might be better to return `AsciiString` on `HttpScheme#name()` like Netty's `HttpScheme#name()`.

Reply:
We can not expose Netty types in our public API, we can only do this for internal utilities.